README.md

```diff
@@ -127,127 +127,175 @@ future state of the libraries and to discover projects for contribution.
-    will become 'approved'.
-13. At some point, someone will pick up the RFC for implementation. For major
-    features this usually requires devising a detailed implementation plan. To
-    that end, submit an **additional PR** on the RFC doc that either fills in
-    the "Implementation Plan" section or references a separate document or
-    GitHub Project Board which includes the plan.
-14. Once this PR is approved, the RFC will move to the 'implementing' state.
-    Usually we track implementation using GitHub projects.
-15. Once implementation is complete, the RFC moves to 'done', and it's issue is
-    closed.
-
-> If the submitter is someone from our CDK community (i.e., not core team member),
-a core team member will be assigned to 'shepherd' each proposal. They will
-generally be the ones updating the RFCs state in the tracking issue as it moves
-through the process. They can decide when a final comment period is triggered.
->
-> On the other hand, if the submitter is a core team member, they will identify
-another core team member, with consent, as their 'shepherd'. The shepherd would
-be the first contact for brainstorming, process and reviews. The core team
-would defer to the shepherd to do the first few rounds of reviews, after which
-the rest of the team should be engaged.
-
-## RFC Life Cycle
+To kick off an RFC process, simply create a [new tracking issue] and follow the
+instructions in the template. The issue includes a checklist of the various
+stages an RFC goes through.
+
+[new tracking issue]: https://github.com/aws/aws-cdk-rfcs/issues/new?assignees=&labels=management%2Ftracking%2C+status%2Fproposed&template=tracking-issue.md&title=proposal+title
+
+This section describes each stage in detail, so you can refer to it for
+guidance.
+
+### 1. Create a tracking issue
+
+Each RFC has a GitHub issue that tracks it from start to finish. The issue is
+the hub for conversations, community signal (+1s) and the issue number is used
+as the unique identifier of this RFC.
+
+> Before creating a tracking issue, please search for similar or related ideas in
+the RFC table above or in the issue list of this repo. If there is a relevant
+RFC, collaborate on that existing RFC, based on its current stage.
+
+Our [tracking issue template] includes a checklist of all the steps an RFC goes
+through and it's the driver's responsibility to update the checklist and assign
+the correct label to on the RFC throughout the process.
+
+[tracking issue template]: https://github.com/aws/aws-cdk-rfcs/blob/master/.github/ISSUE_TEMPLATE/tracking-issue.md
+
+When the issue is created, it is required to fill in the following information:
+
+1. **Title**: the name of the feature or change - think changelog entry.
+2. **Description**: a _short_ description of feature, as if it was already implemented.
+3. **Proposed by**: fill in the GitHub alias of the person who proposed the idea
+   under "Proposed by".
+
+### 2. Identify roles and responsibilities
```

Comment:
As mentioned above, we should describe how to find people for these roles.

Moreover, the first part of this README says that 'anyone' can create an RFC. If so, we need a lot more guidance for contributors not on the team.

.... Or, we reduce the scope and say that RFCs are created by the CDK team. (we would still need some guidance in this paragraph)

Reply:
Modified to reframe as "API Bar Raiser" - let me know what do you think